luigi_tools/parameter.py

```diff
@@ -91,6 +96,12 @@ class PathParameter(luigi.Parameter):
         absolute (bool): the given path is converted to an absolute path.
         create (bool): a folder is automatically created to the given path.
         exists (bool): raise a :class:`ValueError` if the path does not exist.
+
+    .. moved_to_luigi::
+        :previous_luigi_version: 3.0.3
+
+        The moved version dropped the ``create`` parameter as it was confusing for users that may
+        use it instead of using a target.
```

Comment:
Should wait for https://github.com/spotify/luigi/pull/3126 to be merged or closed